helloworld/main.py

```diff
@@ -20,3 +78,8 @@ def hello_http(request):
     else:
         name = 'World'
     return 'Hello {}!'.format(escape(name))
+
+
+if __name__ == '__main__':
+    # change the port number if 4000 doesn't work on your local machine
+    app.run(host='127.0.0.1', port=4000)
```

Comment:
is this here for local dev or something?

Reply:
Yeah, I was not sure how to test my https function locally without first deploying it. So I tried it over the Postman and it worked.

Reply:
that's cool! Might be something @liuhantao9 wants to look into

Reply:
Yeah, I am trying to test my function. There is no route, how do you get http reequest to work? @Abinashi7